ground/src/main/java/com/google/android/ground/MainActivity.kt

```diff
@@ -58,7 +60,9 @@ class MainActivity : AbstractActivity() {
   @Inject lateinit var activityStreams: ActivityStreams
   @Inject lateinit var viewModelFactory: ViewModelFactory
   @Inject lateinit var navigator: Navigator
+  @Inject lateinit var authManager: AuthenticationManager
   @Inject lateinit var userRepository: UserRepository
+  @Inject lateinit var popups: EphemeralPopups
```

Comment:
Can we remove these unused dependencies?

Reply:
Done